embrace-android-core/src/test/java/io/embrace/android/embracesdk/internal/session/orchestrator/SessionOrchestratorTest.kt

```diff
@@ -98,11 +99,13 @@ internal class SessionOrchestratorTest {
         createOrchestrator(true)
         clock.tick()
         val foregroundTime = clock.now()
+        val sessionSpan = currentSessionSpan.sessionSpan
```

Comment:
We were validating the FakeCurrentSessionSpan before which was not modified because the FakeCurrentSessionSpan wasn't patched through. It is now, so the end time will be set properly.